src/backend/distributed/shared_library_init.c

```diff
@@ -693,6 +693,17 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NoticeIfSubqueryPushdownEnabled, NULL, NULL);
 
+	DefineCustomRealVariable(
+		"citus.local_node_parallel_execution_factor",
```

Comment:
This name would be pretty hard for users to understand. What are some alternative names? 

I'm also wondering whether we should just use a number of connections and default to half the connections if set to -1, at least if that leads to a more easily understandable name.

Reply:
I guess you are right, especially considering that we already have `citus.max_shared_pool_size`, it'd be preferable to be inline with that.

What do you think about `citus.local_shared_pool_size'? It might make the explanation easier?